.github/workflows/build-release.yml

```diff
@@ -23,6 +23,16 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+
```

Comment:
Will this displace any "PLAYGROUND" mode code?

Reply:
@leecalcote  yeah the code in ui that was used to lockdown the navigator and header component is not needed now ( i have removed the code for lockdown of header components )